lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -33,7 +33,7 @@ function generateSize(width, height, prefix = 'displayed') {
   return size;
 }
 
-function generateImage(size, coords, networkRecord, src = 'https://google.com/logo.png') {
+function generateImage(size, coords, networkRecord, src = 'https://google.com/logo.png', loading = null) {
```

Comment:
maybe time to convert this to a function with a single object parameter, instead of 5 params. atm you must memorize the order of these params to understand each test case - would be much better as an object, like:

```js
generateImage({size: generateSize(x,y), x: coordx, y: coordy, ...})
```